src/gobimport/converter.py

```diff
@@ -232,6 +232,19 @@ def _extract_source_info(value):
         }
 
 
+def _get_gob_type(typeinfo):
```

Comment:
Probably better to move this into GOB-Core.

Reply:
And then rename it to get_gob_type.

Reply:
Moved to GOB-Core done